pyomo/contrib/doe/doe.py

```diff
@@ -189,10 +187,9 @@ def stochastic_program(self,  design_values, if_optimize=True, objective_option=
         analysis_optimize: result summary of the optimization problem solved
 
         """
-        time0 = time.time()
-
         # store inputs in object
-        self.design_values = design_values
+        self.design_values = design_object.special_set_value
+        self.design_object = design_object
```

Comment:
What information in the original instance of `design_object` was used to initialize the DesignOfExperiments class? What happens if a completely different `design_object` is passed in here? This interface design seems fragile. At a minimum, a lot of error checking is needed.